src/API/src/db/occurrence/occurrence.service.ts

```diff
@@ -17,4 +17,19 @@ export class OccurrenceService {
   findAll(): Promise<Occurrence[]> {
     return this.occurrenceRepository.find({ relations: ['site', 'sample'] });
   }
+
+  async findLocations(
```

Comment:
I know this is a pain, but can we change `Location` to `Occurrence` here and elsewhere?